test/cql-pytest/conftest.py

```diff
@@ -36,6 +36,11 @@ def pytest_addoption(parser):
         help='CQL server port to connect to')
     parser.addoption('--ssl', action='store_true',
         help='Connect to CQL via an encrypted TLSv1.2 connection')
+    # Used by the wrapper script only, not by pytest, added here so it appears
```

Comment:
I will not object to this option if you really want it (it doesn't hurt anybody to have another option), but I  will not use it myself: If the test fails the log is often useful, and it's not saved anywhere else. I normally just run `test/cql-pytest/run |& less`, which is shorter than `test/cql-pytest/run --omit-scylla-output` , and I simply get the whole thing but don't have to view at and can look just at the beginning if I want.